Build bug: Error when running "make docker-mcp"

When I run the `make` command in the documentation, I get an error saying the directory does not exist.

```
user@hostname:~/git/mcp-gateway$ make docker-mcp
CGO_ENABLED=0 go build -trimpath -ldflags "-s -w -X github.com/docker/mcp-gateway/cmd/docker-mcp/version.Version=v0.11.0" -o ./dist/docker-mcp ./cmd/docker-mcp
rm "/home/trevor/.docker/cli-plugins/docker-mcp" || true
rm: cannot remove '/home/trevor/.docker/cli-plugins/docker-mcp': No such file or directory
cp "dist/docker-mcp" "/home/trevor/.docker/cli-plugins/docker-mcp"
cp: cannot create regular file '/home/trevor/.docker/cli-plugins/docker-mcp': No such file or directory
make: *** [Makefile:59: docker-mcp] Error 1
```

Reproduction Steps:

- Clone repository
- Run `make docker-mcp`

--- a/README.md
+++ b/README.md
@@ -51,7 +51,8 @@ The MCP cli will already be installed on recent versions of Docker Desktop but y
 ```bash
 # Clone the repository
 git clone https://github.com/docker/mcp-gateway.git
-cd docker-gateway
+cd mcp-gateway
+mkdir -p "$HOME/.docker/cli-plugins/"
 
 # Build and install the plugin
 make docker-mcp

